src/Silverback.Integration.Kafka/Messaging/KafkaConsumerEndpoint.cs

```diff
@@ -310,8 +309,7 @@ public override void Validate()
 
             if (MaxDegreeOfParallelism < 1)
             {
-                throw new EndpointConfigurationException(
-                    "MaxDegreeOfParallelism must be greater or equal to 1.");
+                throw new EndpointConfigurationException("MaxDegreeOfParallelism must be greater or equal to 1.");
```

Comment:
Since you reformatted, remove the brackets as well, to look like the other ifs.

Reply:
Fixed, see: https://github.com/BEagle1984/silverback/pull/172/commits/62e9b8b415772dfaab2ac3f648be3fb709e33cde